alti/tests/integration/test_profile.py

```diff
@@ -137,3 +157,8 @@ def test_profile_lv03_json_invalid_offset(self):
         params = {'geom': '{"type":"LineString","coordinates":[[550050,206550],[556950,204150],[561050,207950]]}', 'offset': 'asdf'}
         resp = self.testapp.get('/rest/services/profile.json', params=params, headers=self.headers, status=400)
         resp.mustcontain("Please provide a numerical value for the parameter 'offset'")
+
+    def test_profile_entity_too_large(self):
+        params = {'geom': create_json(7000), 'sr': '2056'}
+        resp = self.testapp.get('/rest/services/profile.json', params=params, headers=self.headers, status=413)
+        resp.mustcontain('LineString too large')
```

Comment:
I think we also need to test the same for POST requests.

Reply:
Yes I add a test

Reply:
test added